webapp/src/ts/providers/translation-loader.provider.ts

```diff
@@ -1,11 +1,10 @@
-const translationUtils = require('@medic/translation-utils');
-const DOC_ID_PREFIX = 'messages-';
-
 import { Injectable } from '@angular/core';
 import { TranslateLoader } from '@ngx-translate/core';
 import { from } from 'rxjs';
 
 import { DbService } from '../services/db.service';
+import * as translationUtils from '@medic/translation-utils';
+import { TranslationDocsMatcherProvider } from '@mm-providers/translation-docs-matcher.provider';
 
 @Injectable()
 export class TranslationLoaderProvider implements TranslateLoader {
```

Comment:
Can you please add a dedicated unit test file for this provider with error and success cases?

Reply:
Sure thing :)

Reply:
There already is a unit test: https://github.com/medic/cht-core/blob/4750-angular-10-take-2/webapp/tests/karma/ts/providers/translation-loader.provider.spec.ts
Does it suffice?